boards/arm/actinius_icarus/actinius_icarus_common_2_0_0.dtsi

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2022 Actinius
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default_v2: uart0_default_v2 {
```

Comment:
why "v2"? that means original common is no longer common, so you should likely remove `uart0_default` from common?

Reply:
`v2` is for "v2 hardware" (there are 2 revisions of the Icarus in the definitions now). `common` is for "common between secure and not secure". I hope this clarifies things

Reply:
uart0 of the v2 revision is "overlay-ed" to use this v2 pinctrl, see ref to line 28

Reply:
so there's something wrong: v1 uart is not common and so it should not be in the common file. just have a true common file, a v1 dts and a v2 dts.

Reply:
I am not sure what you see as wrong here; there is a "base" DTS which belongs to the v1.4 rev and an overlay that adjusts for v2. This is exactly the same way it is done in other boards. See for instance nrf9160dk_nrf9160 v14 spi3.
And to re-iterate, "_common" has the meaning of "common between secure and non-secure" in this context.

Reply:
Please check https://github.com/teslabs/zephyr/commit/541d569be322bc3e97d38bfedd89f5424c478f8b